tests/src/volume-smoke.spec.ts

```diff
@@ -0,0 +1,80 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import type { Page } from '@playwright/test';
+import { expect as playExpect } from '@playwright/test';
+import { afterAll, beforeAll, beforeEach, describe, expect, test } from 'vitest';
+import { WelcomePage } from './model/pages/welcome-page';
+import { NavigationBar } from './model/workbench/navigation';
+import { PodmanDesktopRunner } from './runner/podman-desktop-runner';
+import type { RunnerTestContext } from './testContext/runner-test-context';
+
+let pdRunner: PodmanDesktopRunner;
+let page: Page;
+let navBar: NavigationBar;
+
+beforeAll(async () => {
+  pdRunner = new PodmanDesktopRunner();
+  page = await pdRunner.start();
+  pdRunner.setVideoAndTraceName('volume-e2e');
+
+  const welcomePage = new WelcomePage(page);
+  await welcomePage.handleWelcomePage(true);
+  navBar = new NavigationBar(page);
+});
+
+afterAll(async () => {
+  await pdRunner.close();
+});
+
+beforeEach<RunnerTestContext>(async ctx => {
+  ctx.pdRunner = pdRunner;
+});
+
+const volumeName = 'e2eVolume';
+
+describe('Volume workflow verification', async () => {
+  test('Create new Volume', async () => {
+    const volumesPage = await navBar.OpenVolumes();
+    await playExpect(volumesPage.heading).toBeVisible();
+
+    const volumeRow = await volumesPage.getVolumeRowByName(volumeName);
```

Comment:
I'm not sure if I understand the scenario.

When you mean empty page, you mean a volume page without any volume? If so, what that test brings to us?

Reply:
You are assuming in the test that there are no volumes in the initial state, that assumption is not necessarily correct as such the test will fail since it expects for an empty list which would not be empty.

Reply:
I understood now. Thanks @cbr7